### PR TITLE
Add step to configure devicemapper before starting daemon

### PR DESCRIPTION
**Pull request by mbentley**
Signed-off-by: Matt Bentley matt.bentley@docker.com

### CODE REVIEW

File: cs-engine/install.md [hunk 1]
```diff
@@ -63,27 +63,31 @@ to update its RHEL kernel.
     $ sudo yum install docker-engine
     ```
 
-6.  Enable the Docker daemon as a service and start it.
+6.  Configure devicemapper:
+
+    By default, the `devicemapper` graph driver does not come pre-configured in a production ready state. Follow the documented step by step instructions to [configure devicemapper with direct-lvm for production](../../engine/userguide/storagedriver/device-mapper-driver/#/for-a-direct-lvm-mode-configuration) in order to achieve the best performance and reliability for your environment.
```
Comment (joaofnfernandes) [inline cs-engine/install.md]: /ping @cpuguy83 or @mavenugo for a validation that this config is really needed for a production deployment.

**Comment by joaofnfernandes** [comment]
LGTM from a docs standpoint, waiting for @cpuguy83 or @mavenugo to take a look.

File: cs-engine/install.md (hunk 1 above)
Comment (cpuguy83) [inline cs-engine/install.md]: LGTM

**Comment by johndmulhausen** [comment]
Looks like @cpuguy83 LGTMed awhile ago in an inline comment. Thanks @mbentley :)

**Pull request merged**